src/components/ItemList/createItemList.test.js

```diff
@@ -0,0 +1,12 @@
+import createItemList from './createItemList';
+
+describe('test for ItemList component', () => {
```

Comment:
Since you are testing each component in its own separate file, I don't think it is necessary to add a `describe()`.